amplipi/streams/media_device.py

```diff
@@ -63,10 +62,10 @@ def _activate(self, vsrc: int, remake_list: bool = True):
         except Exception as e:
           logger.error(f'Error processing request: {e}')
 
-      if self.song_index < len(self.song_list) and self.device is not None:
+      if self.song_index < len(self.song_list) and utils.virtual_output_device(vsrc) is not None:
```

Comment:
I think you can get rid of ``and utils.virtual_output_device(vsrc) is not None``

Reply:
try it and lmk